.gitignore

```diff
@@ -2,4 +2,8 @@
 .DS_Store
 build/
 flocx_market.egg-info/
-*/__pycache__/*
\ No newline at end of file
+*/__pycache__/*
+.idea/
+.tox/
+AUTHORS/
```

Comment:
We probably don't need to list `AUTHORS` and `ChangeLog` in the `.gitignore` file (unless something is creating them for us automatically?). Also, since they have a trailing `/`, these entries would only match *directories* and would not match files.